mckhannah1901/Co-op-Blog-Project
# Ath 1124 archive blog articles

### JIRA
[ATH-1124-Archive_Blogs](https://wpengine.atlassian.net/browse/ATH-950)

### What has been done
- Posts which are more than 1 week old are put into a separate view (Archived posts)
- Posts which are less than a week old remain in the normal blog posts view

### How to test
- Select blog posts link
- Only posts less than a week old should be displayed
- Select archived posts
- Posts which are more than a week old are displayed here
